### PR TITLE
Issue965 filesystem based schema registry

### LINKED ISSUE #965: Implement filesystem based schemaregistry
The current classpath-based SR is not usable in a test/docker scenario

### PR DESCRIPTION
**Pull request by BernhardBln**
Fixes #965

### CODE REVIEW

**Comment by BernhardBln** [comment]
I activated ComponentScan for `org.factcast.store.pgsql.registry`. 

I wrote unit tests for the file fetcher, and I tried out the whole spring magic once by setting `factcast.store.pgsql.schemaRegistryUrl` to a `file:///` absolute path on my machine and ran ´TransformationTest´.

File: factcast-store-pgsql/README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
 
 PostgreSQL based implementation of a FactStore.  
 
 Uses LISTEN / NOTIFY in order to query with minimum latency and relies on JSONB Columns for indexing/filtering. 
+
+# Schema registries
+
+The following protocols exist for reading schemata: 
+
+- http://...
```
Comment (uweschaefer) [inline factcast-store-pgsql/README.md:11]: plz also add to factcast-site/documentation/content/setup/properties

**Review by uweschaefer — COMMENTED** [review]
LGFF, can we also use this in some integration test ?

File: factcast-store-pgsql/README.md (hunk 1 above)
Comment (BernhardBln) [reply factcast-store-pgsql/README.md:11]: Done, please check.

**Review by uweschaefer — APPROVED** [review]

**Pull request merged**